### PR TITLE
Credential email

### PR DESCRIPTION
**Pull request by lamawmouk**
I wanted to make changes to the code base. This pull request makes two changes to be used on the credentialing pages:

- changing the "Edit Emails" page to prompt users to use an institutional/academic if applicable in the Primary Email section.
- changing the "Application for Credentialed Access" section to prompt users to add an institutional/academic email.

### CODE REVIEW

File: physionet-django/user/templates/user/credential_application.html [hunk 1]
```diff
@@ -16,6 +16,7 @@ <h1>Application for Credentialed Access</h1>
 
   <p>Please use the form below to apply for {{ SITE_NAME }} credentialing. In order to apply:</p>
   <ul>
+    <li>If you have an institutional/academic please add it in your settings (<a href="{% url 'edit_emails' %}">Emails settings</a>).</li>
```
Comment (tompollard) [inline physionet-django/user/templates/user/credential_application.html]: Looks like a word is missing here? "If you have an institutional/academic XXX please add it...".

File: physionet-django/user/templates/user/edit_emails.html [hunk 2]
```diff
@@ -62,7 +62,7 @@ <h5 class="modal-title">Remove Email</h5>
 
 <br>
 <h4>Primary Email</h4>
-<p>The email address used for authentication, and email notifications.</p>
+<p>Please use an institutional/academic email address. This will be used for authentication, and email notifications.</p>
```
Comment (tompollard) [inline physionet-django/user/templates/user/edit_emails.html]: We should probably acknowledge that not everyone will have an institutional/academic email address. We could say something like: 

"The email address used for authentication, and email notifications. We recommend using an institutional/academic email address where possible."

**Review by tompollard — COMMENTED** [review]
Thanks Lama, it's a good idea to update this text. I've added a couple of suggestions.

File: physionet-django/user/templates/user/credential_application.html (hunk 1 above)
Comment (lamawmouk) [reply physionet-django/user/templates/user/credential_application.html]: thanks @tompollard, added the email address to the text.

File: physionet-django/user/templates/user/edit_emails.html (hunk 2 above)
Comment (lamawmouk) [reply physionet-django/user/templates/user/edit_emails.html]: sounds good, added the sentence as part of the edit emails section.

**Comment by tompollard** [comment]
Looks good to me, thanks Lama!

**Pull request merged**